docs/config.md

```diff
@@ -0,0 +1,38 @@
+# Adding an  extra category to the search bar
+
+# To start of here is an example config
+
+		"ImqsServerMirror": {
+			"Tables": {
+			"AssetComponent": {
+					"Categories": [
+						"Assets"
+					],
+					"FriendlyName": "Facility Name",
+					"Fields": [
+						{
+							"FriendlyName": "ComponentID",
+							"Field": "ComponentID",
+							"Order": "1"
+						},
+						{
+							"FriendlyName": "Facility Name",
+							"Field": "AssetFacilityName",
+							"Order": "2"
+						}
+					]
+				},
+
```

Comment:
Surround this with ```json

Reply:
and then a ``` to finish it off. Sorry for two comments, github markdown thing keeps on screwing up my comment

Reply:
Ek kliek nie die een nie

Reply:
` "ImqsServerMirror": {
			"Tables": {
			"AssetComponent": {
					"Categories": [
.....
`
soos dit?